lib_runtime/functoria/functoria_runtime.mli

```diff
@@ -36,11 +36,18 @@ module Arg : sig
 end
 
 val register : 'a Cmdliner.Term.t -> unit -> 'a
+[@@ocaml.deprecated "Use register_arg instead."]
 (** [register t] registers the Cmdliner term [k] as a runtime key and return a
     callback [f] that evaluates to [t]s' value passed on the command-line.
 
     [f] will raise [Invalid_argument] if called before cmdliner's evaluation. *)
 
+val register_arg : 'a Cmdliner.Term.t -> unit -> 'a
+(** [register_arg t] registers the Cmdliner term [k] as a runtime key and return
+    a callback [f] that evaluates to [t]s' value passed on the command-line.
+
+    [f] will raise [Invalid_argument] if called before cmdliner's evaluation. *)
```

Comment:
In the doc string we refer to it as (runtime) key, but we call the function `register_arg`. Should we maybe rename it to `register_key` or update the documentation?

Reply:
I guess we should decide on terminology ;)

I see in the changes entry for 4.5.0 the "configuration key" and "arguments at runtime".

Now, the module is called Key and Runtime_arg. I guess we should stick to that, then.

I find it very subtle, is it `key`, `parameter`, `argument`? Please also note that the docstring was only copied from the `register` function. But I'll adapt to "argument" to stay consistent.